src/android/ForeSeeAPI.java

```diff
@@ -60,48 +62,49 @@ public void onStart() {
         super.onStart();
         if (!Core.isCoreStarted()) {
 
-            this.setListeners();
+            Core.setSDKListener(new CustomVerintSDKListener());
 
             String appId = getAppIdFromJSON();
-            Log.d(sTag, "init the ForeSee SDK");
+            Log.d(logTag, "init the ForeSee SDK");
 
             if (appId != null) {
-                Log.d(sTag, "FCP startup with appId: "+appId);
+                Log.d(logTag, "FCP startup with appId: "+appId);
                 Core.startWithAppId(cordova.getActivity().getApplication(), appId, APP_VERSION);
             } else {
-                Log.d(sTag, "Regular startup");
+                Log.d(logTag, "Regular startup");
                 Core.start(cordova.getActivity().getApplication());
             }
 
-            Core.addCPPValue("crossPlatformName", "Cordova Android");
-            Core.addCPPValue("crossPlatformSDKVersion", CordovaWebView.CORDOVA_VERSION);
-            Core.addCPPValue("crossPlatformOSVersion", android.os.Build.VERSION.RELEASE);
-            Core.addCPPValue("crossPlatformVersion", version);
+            this.addCrossPlatformCPPs();
         }
     }
 
-    void setListeners() {
-        Core.setSDKListener(new CustomVerintSDKListener());
+    void addCrossPlatformCPPs() {
+        Core.addCPPValue("crossPlatformName", "Cordova Android");
+        Core.addCPPValue("crossPlatformSDKVersion", CordovaWebView.CORDOVA_VERSION);
+        Core.addCPPValue("crossPlatformOSVersion", android.os.Build.VERSION.RELEASE);
+        Core.addCPPValue("crossPlatformVersion", version);
```

Comment:
This should be changed to 
```suggestion
        Core.addCPPValue("crossPlatformName", "Cordova Android");
        Core.addCPPValue("crossPlatformSDKVersion", version);
        Core.addCPPValue("crossPlatformOSVersion", android.os.Build.VERSION.RELEASE);
        Core.addCPPValue("crossPlatformVersion", CordovaWebView.CORDOVA_VERSION);
```